### ISSUE TITLE
finalizing ms figures

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/localadaptclim#21]
See also issue #17 ...

@alinazeng Will we also have a map in the main text? If so and it's possible to use the same symbols/colors of the rest of the figures we could refer readers to the map so they can match up the species and gardens. Otherwise we need legends on the scatterplot figures.

**Comment by lizzieinvancouver** [comment]
![Screen Shot 2023-01-24 at 1 01 23 PM](https://user-images.githubusercontent.com/2153556/214286749-3ef214f3-1939-4c87-a402-2fc89652477d.png)
@alinazeng Can you also remind me what the four highlighted points represent for gardens/species? I know I asked in another issue, but I could not quickly find the answer. Thanks!

**Comment by alinazeng** [comment]
Hi Lizzie,

The purple hollow triangles are from Rehfeldt 1994\Picea englemanii. It was a greenhouse experiment in the USA. The green circles are from Rosique-Esplugas 2021\Fraxinus excelsior in the UK. 
Green squares are from Petkova et al., 2017\Fagus sylvatica in Bulgaria. Green hollow triangles are Pinus ponderosa from Dixit et al., 2020, in the USA.

**Comment by lizzieinvancouver** [comment]
@alinazeng Okay! They just seem weird compared to the rest of the data -- but that is often how data looks. Is there anything unusual about them?

Also, is Rehfeldt 1994\Picea englemanii our only greenhouse experiment?

**Comment by alinazeng** [comment]
Hi Lizzie,
@lizzieinvancouver 
I believe Rehfeldt 1994\Picea englemanii is our only greenhouse experiment. I don't really see anything unusual about the way the odds studies were conducted at a quick scan, but there will probably be ways to explain those data points when I read closer. I updated the methods and results section's subheadings and other small things. Could you add to the modeling section at your convenience?

Regarding the next steps, do you think I should go ahead and work on the introduction section?

Thank you.

**Comment by lizzieinvancouver** [comment]
> Regarding the next steps, do you think I should go ahead and work on the introduction section?

Yes! Please start an outline on it (see the writing file for some notes on this [here](https://github.com/temporalecologylab/labgit/blob/master/expectations/writing/tempeco_writing.pdf).

I am printing out the file to work on now, so hopefully will do it next week.
